cmd/controller/state/kubelinter/kubelinter.go

```diff
@@ -20,54 +20,7 @@ import (
-	_ "golang.stackrox.io/kube-linter/pkg/templates/cpurequirements"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/danglinghpa"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/danglingingress"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/danglingnetworkpolicy"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/danglingnetworkpolicypeer"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/danglingservice"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/deprecatedserviceaccount"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/disallowedgvk"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/dnsconfigoptions"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/envvar"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/forbiddenannotation"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/hostipc"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/hostmounts"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/hostnetwork"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/hostpid"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/hpareplicas"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/imagepullpolicy"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/latesttag"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/livenessprobe"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/memoryrequirements"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/mismatchingselector"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/namespace"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/nodeaffinity"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/nonexistentserviceaccount"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/nonisolatedpod"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/ports"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/privileged"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/privilegedports"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/privilegeescalation"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/readinessprobe"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/readonlyrootfs"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/readsecret"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/replicas"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/requiredannotation"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/requiredlabel"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/runasnonroot"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/serviceaccount"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/servicetype"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/sysctl"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/targetport"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/unsafeprocmount"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/updateconfig"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/wildcardinrules"
-	_ "golang.stackrox.io/kube-linter/pkg/templates/writablehostmount"
+	_ "golang.stackrox.io/kube-linter/pkg/templates/all"
```

Comment:
Why this change?

Reply:
We depend on all templates to be loaded because the way we use the registry 🫠 They added new linter checks and I noticed there is this `all` package, which makes our life a lot easier.

Reply:
I think there was some issues and we wanted to exclude some linters which added side effects and more memory usage due to loading something. For trivy that was the case with loading ML model into memory for secrets scanning while we do not use secrets scanning logic.

Reply:
We anyway need to always load all linters, so nothing will change. I switched back to the explicit imports though. A nice benefit is, that we see what new linters have been added.